SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialPlayerSpawner.cpp

```diff
@@ -247,8 +277,16 @@ void USpatialPlayerSpawner::ReceivePlayerSpawnRequestOnServer(const Worker_Comma
 		return;
 	}
 
-	Schema_Object* RequestPayload = Schema_GetCommandRequestObject(Op.request.schema_type);
-	FindPlayerStartAndProcessPlayerSpawn(RequestPayload, Op.caller_worker_entity_id);
+	if (bQueueSpawnRequests)
+	{
+		QueuedPlayerSpawnRequests.Add(TPair<Worker_EntityId_Key, CommandRequestPtr>(
+			Op.caller_worker_entity_id, CommandRequestPtr(Schema_CopyCommandRequest(Op.request.schema_type))));
```

Comment:
I can't find where this is being destroyed. I think this would be a lot safer if using the CommandRequest type instead

Reply:
Ah right it's being done in the unique pointer. Even more reason just to use the wrapping type that already exists.

Reply:
Although I guess that one is just pushing code around as this was one of the attempts to get unique pointers in from before spatial view. So maybe this would really be a driveby

Reply:
I agree - `CommandRequestPtr` is a reimplementation of `OwningCommandRequestPtr` from CommandRequest.h, it'd be best to reuse existing structures.

Reply:
Removed the previous ForwardSpawnRequestDeleter and replaced with OwningCommandRequestPtr

Reply:
As it's a driveby I don't care that much but generally Owning...Ptr types are internal types used to bridge legacy stuff. CommandRequest is the type we want to use as it's harder to misuse. Although I think this correctly uses the OwningPtr.